### PR TITLE
Feature/header contacts

### LINKED ISSUE #33: Create Header component (completed)
### Acceptance criteria's:
User is able to see Header component with the next items inside:
- logo of CoKeeper from *Figma design. Logo consist link, that navigate to the main page and change URL to "/". Depending to the media query (see *SRS "Design", design of logo wil be changed (see *Figma design).
- link to "Contacts" contacts  that navigate to all contact page with URL to "/";
- link to "Favorite" contacts  that navigate to favorite page with URL to "/favorites";
- link to import csv form with icon "CSV" from *Figma design and change URL to "/importCSV";

User is able to change the resolution of the screen and will be provided desktop and mobile view. See details info about breakpoint of the screen resolution (*SRS, design, Figma design).

### Estimation: 4h

**Links:**
Figma design - [Figma](https://www.figma.com/file/J5Syu1z6LPJhIQdScukaa3/CoKeeper?node-id=0%3A1)
SRS - [SRS](https://docs.google.com/document/d/1Z0ceG_NtqwGXs6xRjqq8hb9AoqWj1lLzboL5epCSfkQ/edit#)

### PR DESCRIPTION
**Pull request by solomiia-sahala**
added header component with styling for mobile and desktop resolution, added navigation by buttons

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -5,10 +5,14 @@ import '../styles/Header.scss';
 const logoDesktop = './images/logo-desktop.svg';
 const logoMobile = './images/logo-mobile.svg';
 const iconImportCSV = './images/importCSV.svg';
+const iconHome = './images/home.svg';
+const iconPeopleFavorite = './images/people-favorite.svg';
 
 const Header = () => {
+    const url = window.location.href.split('/');
```
Comment (chiffa-fox) [inline src/components/Header.tsx]: why not use 
`window.location.pathname` ?

File: src/components/Header.tsx [hunk 2]
```diff
@@ -18,6 +22,18 @@ const Header = () => {
                         </picture>
                     </a>
                 </div>
+                <div className="contacts">
+                    <a href="/" className={`${currentPath === '' ? 'line' : ''}`}>
```
Comment (chiffa-fox) [inline src/components/Header.tsx]: I'd suggest considering the usage of 'classnames' library. It is quite popular and works perfectly for situations like yours when you need to add a class or nothing

Comment (solomiia-sahala) [reply src/components/Header.tsx]: thanks for the advice, will update the request with usage of this library.

File: src/components/Header.tsx (hunk 1 above)
Comment (solomiia-sahala) [reply src/components/Header.tsx]: this is exactly perfect decision  for my case, thank you.

**Review by chiffa-fox — APPROVED** [review]

File: src/components/Header.tsx (hunk 2 above)
Comment (alexandrbig) [reply src/components/Header.tsx]: https://reactrouter.com/web/api/NavLink/activeclassname-string

**Pull request merged**